visualize_ta.py

```diff
@@ -0,0 +1,122 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+import wandb
+
+from typing import List
+
+from src.utils.data_transform import PlanetTransform
+
+
+def get_ndvi(image_stack):
+    PlanetTransform.per_band_mean
```

Comment:
What is it doing standalone?

Reply:
This is a helper function for making these plots
<img width="1332" alt="image" src="https://user-images.githubusercontent.com/25067660/157383926-467219ea-9315-4e5f-8f9d-04adf82f617c.png">